packages/protocol/contracts/tokenvault/ERC20Vault.sol

```diff
@@ -355,7 +360,17 @@ contract ERC20Vault is BaseVault {
         // If it's a bridged token
         if (bridgedToCanonical[_token].addr != address(0)) {
             ctoken_ = bridgedToCanonical[_token];
-            IBridgedERC20(_token).burn(msg.sender, _amount);
+
+            bytes[] memory encodedCalls = _getBurnLowLevelCalls(_token, _user, _amount);
```

Comment:
I prefer to handle all Bridged token the same way if possible. Rather than burning the token, I think transfer the token to this address then burn it is a much better idea.